config/default.json

```diff
@@ -1,5 +1,12 @@
 {
   "App": {
-    "port": 8080
+    "port": 8080,
+    "host": "localhost",
+    "database": {
+      "host": "postgres",
```

Comment:
This configuration assuming that we will using only one DB which isn't true ;) moreover instead of splitting it into user, password etc. you should use connectionString which will come from environment variables. Plese read how to use node config for environment variables.